frcxx_nft/src/types.rs

```diff
@@ -64,7 +115,16 @@ pub struct TransferParams {
 
 impl Cbor for TransferParams {}
 
-#[derive(Serialize_tuple, Deserialize_tuple, Debug)]
+#[derive(Serialize_tuple, Deserialize_tuple, Clone, Debug)]
+pub struct TransferReturn {
+    pub from_balance: u64,
+    pub to_balance: u64,
+    pub token_ids: Vec<TokenID>,
```

Comment:
Document which token ids are here. The easy answer is the ones that were attempted to be transferred, but the receiver hook could have burnt, transferred elsewhere, or returned them, or even minted new ones.

I think the easy answer could be ok, but document that this might be inconsistent with the balances (which will be snapshots that do take into account any hook-related changes).

Reply:
https://github.com/helix-onchain/filecoin/issues/158